src/templates/home-page.js

```diff
@@ -25,24 +26,41 @@ export const HomePageTemplate = ({
   advisors_section: advisorsSection = {},
   supporters_section: supportersSection = {},
 }) => {
-  const handleSignupDiscord = ({ email }) => {
-    signupMailingList({ email, discordSignup: true })
-  }
-
   return (
     <div className="main-content">
-      <PageSection id={sections.HOME}>
+      <PageSection
+        id={sections.HOME}
+        style={{
+          backgroundImage: `url(${hero.bg_image.childImageSharp.fluid.src})`,
+        }}
+      >
         <Row>
-          <Col xs={12} sm={7}>
-            <h1>{hero.title}</h1>
+          <Col xs={12}>
+            <h1 dangerouslySetInnerHTML={{ __html: hero.title }} />
             <div
               className="body"
               dangerouslySetInnerHTML={{ __html: hero.body }}
             />
           </Col>
-          <Col xs={12} sm={5} className="col-circles">
-            <Image imageData={images.textureCircle1} />
-          </Col>
+        </Row>
+        <Row className="cta-section">
+          {hero.cta ? (
+            <Col xs={12} sm={12} md={6} lg={4} className="cta">
+              {hero.cta.icon ? <Image imageData={hero.cta.icon} /> : ""}
+              <h2>{hero.cta.label}</h2>
+            </Col>
```

Comment:
Structure-wise, I'll say it would make more sense for this to be along the lines of:

```
<h2 class="cta col-12 col-sm-12 col-md-6 col-lg-4">
  {hero.cta.icon ? ...}
  {hero.cta.label}
</h2>
```

I know some of the positioning gets complicated though, so no worries with sticking to this for now :)

Reply:
Going to hold off on this one for now...to change it this way, my instinct is to add a `<span>` around the label text which seems to defeat the purpose of this note 😅 ...maybe we can futz with the widths later if not